package.json

```diff
@@ -41,7 +41,9 @@
     "test:e2e": "jest '(\\/test\\/e2e/).*'",
     "pack": "npm run build && electron-builder --dir",
     "dist": "npm run build && electron-builder",
-    "postinstall": "electron-builder install-app-deps"
+    "postinstall": "electron-builder install-app-deps",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
```

Comment:
you missed prestorybook task to build story book
`"prestorybook": "yarn run build-storybook"`

For the sake of consistency you might wanna rename build-storybook to build:storybook